tests/models/lxmert/test_modeling_tf_lxmert.py

```diff
@@ -581,73 +580,6 @@ def test_compile_tf_model(self):
             extended_model = tf.keras.Model(inputs=[input_ids, visual_feats, visual_pos], outputs=[outputs])
             extended_model.compile(optimizer=optimizer, loss=loss, metrics=[metric])
 
-    @tooslow
-    def test_saved_model_creation(self):
-        pass
-
-    @slow
-    def test_saved_model_creation_extended(self):
```

Comment:
Does this run now with the default `test_saved_model_creation_extended` test?

Reply:
`test_saved_model_creation_extended` is now a core test that is only run on a few models because it's very expensive, so this skip is no longer needed.